src/components/tokens/TokenAction.js

```diff
@@ -134,6 +134,7 @@ class TokenAction extends React.Component {
     return (
       <div key={this.props.action}>
         <h2>{this.props.title}</h2>
+        <p>{this.props.subtitle}</p>
```

Comment:
I think we should calculate the deposit in real-time and show to the user.
For example:
It the user typed 120 we would write "A deposit of 1.20 HTR will be made".

Reply:
Did that and updated the pictures. Can you review again?

Reply:
Great. Thanks!